prediction_market_agent/agents/safe_guard_agent/streamlit_pages/demo_page.py

```diff
@@ -99,6 +99,7 @@ def get_safe_history_multisig_cached(
                 [tx.id for tx in historical_transactions],
                 index=None,
                 disabled=queue_transaction_id is not None,
+                accept_new_options=True,
```

Comment:
Allow to copy paste any tx id into the selector in the web app, why history endpoint doesn't return it is still to be figured out: https://github.com/gnosis/prediction-market-agent/issues/871